atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlExecutorImpl.java

```diff
@@ -125,18 +183,9 @@ private CqlResult executeQueryOnHost(CqlQuery cqlQuery, InetSocketAddress host)
             try {
                 return clientPool.runWithRetryOnHost(host, createCqlFunction(cqlQuery));
             } catch (UnavailableException e) {
-                throw wrapIfConsistencyAll(e);
+                throw new InsufficientConsistencyException("This query requires consistency " + consistency);
```

Comment:
We should have the `UnavailableException` as the cause here. Also, can we log the `CqlQuery.queryFormat` as `This query` is quite vague.